### ISSUE TITLE
Feature Request: Lock Miniview w/ Mouse Clickthrough.

### ISSUE THREAD

**Issue by TheYokai** [iamlemec/miniview#10]
It would be nice if the user could lock the current position / opacity of the miniview and then allow the user to click through the window when you mouse over. This way, the miniview would be more useful in circumstances where you may need all of the screen space (Specifically in programs like blender or krita).

**Comment by iamlemec** [comment]
This would be pretty useful. I'm not entirely sure it's possible though. Getting full transparency on mouseover would be straightforward, but getting clickthrough would require disabling the window, meaning we would never get a leave-event from the mouse. I'm open to ideas though.

**Comment by Eoin-ONeill-Yokai** [comment]
Yes, I think I get what you mean. I will look a little bit into seeing if there's a way for X to treat a given window as a mouse-transparent window (where it ignores mouse inputs all-together.)

For the time being, maybe instead it could be 'minimized' on an initial click and become visible again after the mouse leaves the original screen space of the mini-window? Some sort of solution would be nice for those of us with limited screen real estate.
